lib/home_page.dart

```diff
@@ -38,76 +36,107 @@ class _HomePageState extends State<HomePage> {
+              return SplashScreen();
+            } else {
+              return Scaffold(
+                backgroundColor: Colors.white,
+                appBar: AppBar(
+                  automaticallyImplyLeading: false,
+                  backgroundColor: Colors.red.shade900,
+                  title: Image.asset(
+                    StringConstants.logoImage,
+                    width: NumericConstants.logoImageWidth,
+                  ),
+                  centerTitle: true,
+                ),
+                body: PokemonState(
+                  stateModel: snapshot.data!,
+                  pokemonBloc: _pokemonBloc,
+                ),
+                bottomNavigationBar: Container(
+                  color: Colors.red.shade900,
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceAround,
+                    children: [
+                      ElevatedButton(
+                        style: ElevatedButton.styleFrom(
+                          primary: Colors.red.shade700,
+                        ),
+                        onPressed: pageNumber != 1
+                            ? () {
+                                _pokemonBloc.getPreviousPage();
+                                pageNumber--;
+                                setState(() {});
+                              }
+                            : () {},
+                        child: Icon(
+                          Icons.arrow_back,
+                          color: Colors.black,
+                        ),
+                      ),
+                      snapshot.data?.state == StateType.loading
+                          ? Text('')
+                          : Stack(
+                              children: [
+                                Text(
+                                  StringConstants.pageText +
+                                      ' $pageNumber - $cantPages',
+                                  style: TextStyle(
+                                    fontFamily: 'Pokemon',
+                                    fontSize: NumericConstants.pageTextFontSize,
+                                    foreground: Paint()
+                                      ..style = PaintingStyle.stroke
+                                      ..strokeWidth = NumericConstants
+                                          .pageForegroundStrokeWidth
+                                      ..color = firstPageColor,
+                                  ),
+                                ),
+                                Text(
+                                  StringConstants.pageText +
+                                      ' $pageNumber - $cantPages',
+                                  style: TextStyle(
+                                    fontFamily: 'Pokemon',
+                                    fontSize: NumericConstants.pageTextFontSize,
+                                    color: secondPageColor,
+                                  ),
```

Comment:
the TextStyles can be added into a styles file

Reply:
Fixed